File: be/src/main/java/com/project/popupmarket/dto/admin/AdminPeriodAnalyticsTO.java
package com.project.popupmarket.dto.admin;

import lombok.*;

import java.util.List;
import java.util.Map;

@Data
@Getter
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class AdminPeriodAnalyticsTO {
    private List<Map<String, Object>> current;
    private List<Map<String, Object>> prev;
    private AdminDataAnalyticsTO data;
}

File: be/src/main/java/com/project/popupmarket/dto/admin/AdminPeriodAnalyticsTO.kt
package com.project.popupmarket.dto.admin

data class AdminPeriodAnalyticsTO(
    val current: List<Map<String, Any>> = emptyList(),
    val prev: List<Map<String, Any>> = emptyList(),
    val data: AdminDataAnalyticsTO = AdminDataAnalyticsTO()
)
